cmsplugin_articles_ai/management/commands/publish_test_articles.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from cmsplugin_articles_ai.factories import TaggedArticleFactory
```

Comment:
I suppose this management command should be trashed now that the factories aren't available in installed versions of the package.